editor/src/reveal/editor/reveal.clj

```diff
@@ -65,14 +67,24 @@
 
 (defn- node-children-fn [ec node-id]
   (fn []
-    (let [node-type-def @(g/node-type* (:basis ec) node-id)]
-      (->> [:input :property :output]
-           (mapcat (fn [k] (keys (get node-type-def k))))
-           distinct
-           sort
-           (map
-             (fn [label]
-               (label-tree-node ec node-id label)))))))
+    (let [{:keys [basis]} ec
+          node-type-def @(g/node-type* basis node-id)
+          override-original (g/override-original basis node-id)
+          children (->> [:input :property :output]
+                        (mapcat (fn [k] (keys (get node-type-def k))))
+                        distinct
+                        sort
+                        (map
+                          (fn [label]
+                            (label-tree-node ec node-id label))))]
+      (cond->> children
+               override-original
+               (cons {:value override-original
+                      :render (r/horizontal
+                                (r/raw-string "override-original" {:fill :symbol})
+                                r/separator
+                                (node-id-sf ec override-original))
+                      :children (node-children-fn ec override-original)})))))
```

Comment:
I added `override-original` to the node tree view so it's easier to navigate to the original nodes (I needed this when exploring how overrides work when we override materials in models)